test/integration/serviceaccount/legacy_service_account_token_clean_up_test.go

```diff
@@ -49,45 +49,6 @@ const (
 
 func TestLegacyServiceAccountTokenCleanUp(t *testing.T) {
 	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, kubefeatures.LegacyServiceAccountTokenCleanUp, true)()
-	ctx, cancel := context.WithCancel(context.Background())
-	defer cancel()
-
-	fakeClock := testingclock.NewFakeClock(time.Now().UTC())
-
-	c, config, stopFunc, informers, err := startServiceAccountTestServerAndWaitForCaches(ctx, t)
-	defer stopFunc()
-	if err != nil {
-		t.Fatalf("failed to setup ServiceAccounts server: %v", err)
-	}
-
-	// start legacy service account token cleaner
-	startLegacyServiceAccountTokenCleaner(ctx, c, fakeClock, informers)
-
-	// wait configmap to label with tracking date
-	if err := wait.PollImmediate(time.Second, 10*time.Second, func() (bool, error) {
-		configMap, err := c.CoreV1().ConfigMaps(metav1.NamespaceSystem).Get(ctx, legacytokentracking.ConfigMapName, metav1.GetOptions{})
-		if err != nil {
-			return false, err
-		}
-		_, exist := configMap.Data[legacytokentracking.ConfigMapDataKey]
-		if !exist {
-			return false, fmt.Errorf("configMap does not have since label")
-		}
-		return true, nil
-	}); err != nil {
-		t.Fatalf("failed to wait configmap starts to track: %v", err)
-	}
```

Comment:
keep the server and cleaner common, just move the namespace and service account inside the test case

Reply:
hmm... starting a server is fairly expensive... I don't really want to do that for every testcase if we can avoid it.

can we reconstruct the fakeClock, cleaner, and a partitioned namespace for each testcase?

Reply:
Thank you very much, Jordan! I am so stupid that I did not realized I don't need to make the server for each test case. I have revised that.